### ISSUE TITLE
Error messages should tell user how to download logs when filing an issue

### ISSUE THREAD

**Issue by nephix** [raiden-network/light-client#1131]
## Description

Currently, error messages tell the user sometimes to copy logs from console and file a new issue.

Instead, they should tell the user to download logs from splash screen and upload them to the issue

## Acceptance criteria
- Bug template is adjusted to make logs mandatory
- Locales tell user to download logs in correct spot

## Tasks
- [ ]

**Comment by nephix** [comment]
Should be done after #152 because the location of the log download link will probably change

**Comment by nephix** [comment]
I'll remove this issue from the current iteration, as the depending ticket is not in this iteration as well
